### PR TITLE
Converted ReadyForReviewComponent to function

### PR DESCRIPTION
**Pull request by Eezi**

### CODE REVIEW

File: packages/client/src/views/OfficeHome/readyForReview/ReadyForReview.tsx [hunk 1]
```diff
@@ -82,87 +82,81 @@ interface IBaseReviewTabProps {
...
 
-  recordWindowWidth = () => {
-    this.setState({ width: window.innerWidth })
-  }
+  useEffect(() => {
+    window.addEventListener('resize', recordWindowWidth)
+    return () => {
+      window.removeEventListener('resize', recordWindowWidth)
+    }
+  }, [])
```
Comment (Zangetsu101) [inline packages/client/src/views/OfficeHome/readyForReview/ReadyForReview.tsx:114]: @Eezi Can we extract this out to a custom hook please? I'm pretty sure there are other components which will be using this

**Review by Zangetsu101 — COMMENTED** [review]
Great work as always!

File: packages/client/src/views/OfficeHome/readyForReview/ReadyForReview.tsx (hunk 1 above)
Comment (Eezi) [reply packages/client/src/views/OfficeHome/readyForReview/ReadyForReview.tsx:114]: Good idea! I can do that as well.

File: packages/components/src/AppBar/AppBar.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import styled from 'styled-components'
 import { grid } from '../grid'
 import { Stack } from '../Stack'
 import { Text } from '../Text'
-import { useWindowSize } from './useWindowSize'
+import { useWindowSize } from '@opencrvs/client/src/hooks/useWindowSize'
```
Comment (Zangetsu101) [inline packages/components/src/AppBar/AppBar.tsx:16]: client uses components as a dependency so we can't have components importing something from client.

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/components/src/AppBar/AppBar.tsx (hunk 2 above)
Comment (Zangetsu101) [reply packages/components/src/AppBar/AppBar.tsx:16]: The tests are also failing because of this

Comment (Eezi) [reply packages/components/src/AppBar/AppBar.tsx:16]: Should we also add the hooks directory to the components package and then move the useWindowSize hook there and use it in the client package?

**Review by Zangetsu101 — APPROVED** [review]

File: packages/components/src/Hooks/index.ts
Comment (Zangetsu101) [inline packages/components/src/Hooks/index.ts:1]: @Eezi Can we just rename this folder to be `src/hooks` instead please? We kinda use PascalCase for components and this is more like a util so

**Comment by Zangetsu101** [comment]
I think we can merge it in after the hooks folder rename :smiley:

File: packages/components/src/Hooks/index.ts
Comment (Eezi) [reply packages/components/src/Hooks/index.ts:1]: Yea sure! I will rename this asap.

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**